e2e/support/cypress_data.js

```diff
@@ -7,7 +7,7 @@ import { ORDERS_PRODUCTS_ACCESS } from "./test_roles";
  * For that reason, we have some sanity checks in the `default.cy.snap.js` spec.
  *
  * SAMPLE_DB_TABLES contains only the references to the four main tables ids in sample database.
- * We need these references to avoid circular dependecy issue in custom commands and e2e helpers.
+ * We need these references to avoid circular dependency issue in custom commands and e2e helpers.
```

Comment:
Maybe it's time to revisit adding [CodeSpell](https://github.com/codespell-project/codespell) to CI

Reply:
I use vscode extension locally to fix those issues. Definitely wrong spelling is not the biggest problem nowadays, I just stumbled upon this one after searching for `circular dep` phrase 🙂